README.md
# this is a website im making that will contain :

### memez at <a href="https://git.io/memez">git.io/memez</a>

### and some gamez i made and will make at <a href="https://git.io/gamez">git.io/gamez</a>

### and also thejunkyard at <a href="https://git.io/thejunkyard">git.io/thejunkyard</a> which i want it to be a board of guud b-stuff like b-games and shit and some other dope guud shit too.

### and a polished website too at <a href="https://git.io/fancystuff">git.io/fancystuff</a>


CHANGELOG.md
# Changelog

## 0.0.0
- big thingy `l3iba` [linkname](https://github.com/mrwandev): yeeeeee
    - smol thingu
<!-- *Note: a note.* -->

main.js
function bars(x) {
    x.classList.toggle("change");
};

function goBack() {
    window.history.back();
};

function timer(time) {
  const minutes = Math.floor(time / 60);
  let seconds = time % 60;
  if (seconds < 10)
    seconds = `${seconds}`;
 // seconds = `0${seconds}`;

//return `${minutes}:${seconds}`;
  return `${seconds}`;
};

let timerInterval = null;
let timePassed = 0;
var secondsStr;

export function redirectWithTimer(_time, direction, directionStr) {
  const time = _time + 1;
  let timeLeft = time;
  timerInterval = setInterval(() => {
    
    // The amount of time passed increments by one
    timePassed = timePassed += 1;
    timeLeft = time - timePassed;

    if(direction == "")
      direction = "http://mrwandev.github.io/junkyard";

    if(directionStr == "")
          directionStr = "the junkyard";
        
    if(timeLeft == 0)
    	window.location.replace(direction);
    
    if(timeLeft > 1 || timeLeft == 0) {
    	secondsStr = " seconds";
    }
    else if (timeLeft == 1){
    	secondsStr = " second";
    }

	document.getElementById("text").innerHTML = "u will be redirected to " + directionStr + " in " + timer(timeLeft) + secondsStr;
  }, 1000);
};

gamez/akinator_webgl/TemplateData/UnityProgress.js
function UnityProgress(unityInstance, progress) {
  if (!unityInstance.Module)
    return;
  // if (!unityInstance.logo) {
  //   unityInstance.logo = document.createElement("div");
  //   unityInstance.logo.className = "logo " + unityInstance.Module.splashScreenStyle;
  //   unityInstance.container.appendChild(unityInstance.logo);
  // }
  if (!unityInstance.progress) {    
    unityInstance.progress = document.createElement("div");
    unityInstance.progress.className = "progress " + unityInstance.Module.splashScreenStyle;
    unityInstance.progress.empty = document.createElement("div");
    unityInstance.progress.empty.className = "empty";
    unityInstance.progress.appendChild(unityInstance.progress.empty);
    unityInstance.progress.full = document.createElement("div");
    unityInstance.progress.full.className = "full";
    unityInstance.progress.appendChild(unityInstance.progress.full);
    unityInstance.container.appendChild(unityInstance.progress);
  }
  unityInstance.progress.full.style.width = (100 * progress) + "%";
  unityInstance.progress.empty.style.width = (100 * (1 - progress)) + "%";
  if (progress == 1)
    unityInstance.logo.style.display = unityInstance.progress.style.display = "none";
}

javascript.js
function bars(x) {
    x.classList.toggle("change");
};

function goBack() {
    window.history.back();
};

function timer(time) {
  const minutes = Math.floor(time / 60);
  let seconds = time % 60;
  if (seconds < 10)
    seconds = `${seconds}`;
 // seconds = `0${seconds}`;

//return `${minutes}:${seconds}`;
  return `${seconds}`;
};

let timerInterval = null;
let timePassed = 0;
var secondsStr;

function redirectWithTimer(_time, direction, directionStr) {
    const time = _time + 1;
    let timeLeft = time;
    timerInterval = setInterval(() => {
        
    // The amount of time passed increments by one
    timePassed = timePassed += 1;
    timeLeft = time - timePassed;

    if(direction == "")
      direction = "http://mrwandev.github.io/junkyard";

    if(directionStr == "")
        directionStr = "the junkyard";
        
    if(timeLeft == 0)
    	window.location.replace(direction);
    
    if(timeLeft > 1 || timeLeft == 0) {
    	secondsStr = " seconds";
    }
    else if (timeLeft == 1){
    	secondsStr = " second";
    }

	document.getElementById("text").innerHTML = "u will be redirected to " + directionStr + " in " + timer(timeLeft) + secondsStr;
  }, 1000);
};

function usePassword(usePasswordBool, previousBodyColor){
    var usePasswordBool;
    var password = "mrwan69";
    var entredPass = "";

    var body;

    var previousBodyColor;

    if(previousBodyColor == "")
        previousBodyColor = "white";

    if(usePasswordBool){
        var blockPanel = document.createElement("div");
        blockPanel.setAttribute('style', 'width: 100%; height: 100%; background-color: #131313; position: fixed;');
        blockPanel.setAttribute('id', 'block');

        var paragraph = document.createElement("p");
        paragraph.setAttribute('id', 'para');
        paragraph.setAttribute('style', 'color: white');
        var text = document.createTextNode("enter password");
        paragraph.appendChild(text);
        blockPanel.prepend(paragraph);

        var passwordInput = document.createElement("input");
        passwordInput.setAttribute('id', 'passwordInputID');
        passwordInput.setAttribute('type', 'password');

        var body = document.getElementById("bodyy");
        body.setAttribute('style', 'background-color: #131313');
        body.prepend(blockPanel);
        blockPanel.appendChild(paragraph);
        blockPanel.appendChild(passwordInput);

        update = setInterval(() => {

            var body = document.getElementById("bodyy")
            var blockPanel = document.getElementById("block");

            if(blockPanel == null)
            {
                var blockPanel = document.createElement("div");
                blockPanel.setAttribute('style', 'width: 100%; height: 100%; background-color: #131313; position: fixed;');
                blockPanel.setAttribute('id', 'block');
                body.prepend(blockPanel);
            }
            else
                blockPanel = document.getElementById("block");

            var paragraph = document.getElementById("para");

            if(paragraph == null)
            {
                var paragraph = document.createElement("p");
                paragraph.setAttribute('id', 'para');
                paragraph.setAttribute('style', 'color: white');
                var text = document.createTextNode("enter password");
                paragraph.appendChild(text);
                blockPanel.prepend(paragraph);
            }
            else
                paragraph = document.getElementById("para");

            var passwordInput = document.getElementById("passwordInputID");

            if(passwordInput == null)
            {
                var passwordInput = document.createElement("input");
                passwordInput.setAttribute('id', 'passwordInputID');
                passwordInput.setAttribute('type', 'password');
                blockPanel.prepend(paragraph);
                blockPanel.appendChild(passwordInput);
            }
            else
                entredPass = document.getElementById("passwordInputID").value;

            if(entredPass == password){
                body.style.backgroundColor = previousBodyColor;
                body.removeChild(blockPanel);
                clearInterval(update);
            }
        }, 0);
    }
};